solr/modules/ltr/src/java/org/apache/solr/ltr/feature/FieldValueFeature.java

```diff
@@ -166,18 +167,23 @@ public FeatureScorer scorer(LeafReaderContext context) throws IOException {
     public class FieldValueFeatureScorer extends FeatureScorer {
 
       LeafReaderContext context = null;
+      StoredFields storedFields = null;
```

Comment:
Wondering if this could be private and/or final (even though `context` isn't currently)?
```suggestion
      private final StoredFields storedFields;
```

Reply:
> ... (even though `context` isn't currently)?

#1640 proposes to mark `context` private and final.

Reply:
https://github.com/apache/solr/pull/1557/commits/06061e30cd755add6b028dd94a1480a4c941479c to mark private and final.